index.html

```diff
@@ -112,15 +112,18 @@ <h4>Property template</h4>
 
             <div class="card col-6">
                 <header>Property Template</header>
-                <div class="deer-view" title="Sample Prop" deer-key="name" deer-id="http://devstore.rerum.io/v1/id/5c9d155fe4b0a44e13e61706" deer-template="prop"></div>
+                <deer-view title="Sample Prop" deer-key="name" deer-id="http://devstore.rerum.io/v1/id/5c9d155fe4b0a44e13e61706" deer-template="prop"></deer-view>
```

Comment:
This does not work when deer-id="http://devstore.rerum.io/v1/id/11111", I think when the object does not have the prop you are looking for there is trouble.

Reply:
The behavior is that the `id` is used as a value when nothing is available... This can be changed to some filler text or return an error, if we prefer. I checked with this one and it seemed to work.